applications/volume_rendering_xr/main.cpp

```diff
@@ -152,11 +242,37 @@ class App : public holoscan::Application {
 
     add_flow(xr_transform_controller,
              xr_transform_renderer,
+             {{"ux_box", "ux_box"}, {"ux_cursor", "ux_cursor"}, {"ux_window", "ux_window"}});
+
+#define SOURCE_SINK 1
```

Comment:
This can be removed together with the SharedDataSourceOp. This was a workaround for the missing support of graph loops. Since we now depend on Holoscan SDK 2.0 which supports loops this is obsolete.

Reply:
Observed a runtime segfault when attempting to remove source/sink ops from `volume_rendering_xr`:
```
Caught signal 11 (Segmentation fault: address not mapped to object at address 0x74ba6c317310)
 INFO [client_loop] Client disconnected.
./run: line 83:  2295 Segmentation fault      (core dumped) /workspace/holohub/build/applications/volume_rendering_xr/volume_rendering_xr --config /workspace/holohub/data/volume_rendering_xr/config.json --density /workspace/holohub/data/volume_rendering_xr/highResCT.mhd --mask /workspace/holohub/data/volume_rendering_xr/smoothmasks.seg.mhd
```

Given the existing app is functional, let's move forward as-is. I'll create a ticket to revisit and investigate.